print/nuxt.config.js

```diff
@@ -73,47 +78,30 @@ export default {
    ** See https://axios.nuxtjs.org/options
    */
   axios: {
-    baseURL: process.env.INTERNAL_API_ROOT_URL || 'http://backend:3001/api',
+    baseURL: process.env.INTERNAL_API_ROOT_URL || 'http://caddy:3001/',
+    credentials: true,
   },
   /*
    ** Sentry module configuration
    ** See https://sentry.nuxtjs.org/sentry/options
    */
   sentry: {
     // Use a dummy DSN so that the sentry module doesn't disable itself during build
```

Comment:
can the comment be deleted?

Reply:
Yes, the comment is obsolete, will remove.
I didn't check though if sentry still works. Added this to #2481